packages/web3torrent/src/clients/web3t-channel-client.ts

```diff
@@ -0,0 +1,193 @@
+import {ChannelResult, Message, ChannelClient} from '@statechannels/channel-client';
+import {bigNumberify} from 'ethers/utils';
+
+interface ChannelState {
+  channelId: string;
+  turnNum: string;
+  challengeExpirationTime;
+  aUserId: string;
+  bUserId: string;
+  aAddress: string;
+  bAddress: string;
+  aOutcomeAddress: string;
+  bOutcomeAddress: string;
+  aBal: string;
+  bBal: string;
+}
```

Comment:
This could probably be a simpler data structure based on the nature of the Web3Torrent application.

I think every channel has a distinct `sender` and `receiver`, so we could use that instead of `a` and `b`. Then `aBal` and `bBal` could be stored as `amountSent` and `totalBudget` or something like that. I think `challengeExpirationTime` also can just be hard-coded for this application inside the Web3Torrent application code (i.e., in this file as a constant).

If we used the optimal strategy I mentioned, `turnNum` could always just either be `0` or `1`, too.